cli/src/semgrep/metrics.py

```diff
@@ -152,8 +152,7 @@ def suppress_errors(func: Callable[..., None]) -> Callable[..., None]:
     def wrapper(*args: Any, **kwargs: Any) -> Any:
         try:
             return func(*args, **kwargs)
-        except Exception as e:
-            logger.error(f"Error in {func.__name__}: {e}")
+        except Exception:
```

Comment:
```suggestion
        except Exception as e:
            logger.debug(f"Error in {func.__name__}: {e}")
```

This would remain useful for development and for debugging customer issues possibly, so maybe let's just reduce the log level so that it stays visible in logs?